### PR TITLE
feat: Express Checkout beta

### PR DESCRIPTION
**Pull request by semirp**

### CODE REVIEW

**Review by NQuinn27 — APPROVED** [review]

**Review by NQuinn27 — DISMISSED** [review]

**Review by FlaviuExtPrimer — DISMISSED** [review]

File: packages/example/src/screens/SettingsScreen.tsx [hunk 1]
```diff
@@ -104,7 +119,7 @@ const SettingsScreen = ({ navigation }) => {
                 <SegmentedControl
                     testID="Enviroment"
                     style={{ marginTop: 6 }}
-                    values={['Dev', 'Sandbox', 'Staging', 'Production']}
+                    values={['Sandbox', 'Dev', 'Staging', 'Production']}
```
Comment (borisprimer) [inline packages/example/src/screens/SettingsScreen.tsx:122]: Why this?

Comment (borisprimer) [reply packages/example/src/screens/SettingsScreen.tsx:122]: Did I do this? @semirp

File: packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerOrder.kt [hunk 2]
```diff
@@ -1,6 +1,11 @@
 package com.primerioreactnative.extensions
 
 import com.primerioreactnative.datamodels.PrimerOrderRN
+import com.primerioreactnative.datamodels.PrimerShippingRN
 import io.primer.android.domain.action.models.PrimerOrder
+import io.primer.android.domain.action.models.PrimerShipping
 
-internal fun PrimerOrder.toPrimerOrderRN() = PrimerOrderRN(countryCode)
+// TODO enable shipping
```
Comment (borisprimer) [inline packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerOrder.kt:8]: leftover?

**Review by borisprimer — DISMISSED** [review]
lgtm, few nits

File: packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerOrder.kt (hunk 2 above)
Comment (semirp) [reply packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerOrder.kt:8]: no, we have to enable this on native first on our side

File: packages/example/src/screens/SettingsScreen.tsx (hunk 1 above)
Comment (semirp) [reply packages/example/src/screens/SettingsScreen.tsx:122]: it was gabor, let me revert

**Review by FlaviuExtPrimer — APPROVED** [review]

**Pull request merged**